source/helpers/conventional-commits.test.ts

```diff
@@ -0,0 +1,21 @@
+import {test, assert} from 'vitest';
+
+import {parseConventionalCommit} from './conventional-commits.js';
+
+function assertRegexDeepStrictEqual(input: string, expected: [string, string | undefined]) {
+	const result = parseConventionalCommit(input);
+	assert.deepStrictEqual(result, [input, ...expected]);
```

Comment:
When you adjust the expected arrays, this should also be changed:

```suggestion
	assert.deepStrictEqual(result, expected);
```